vic/drivers/shared_all/src/put_data.c

```diff
@@ -540,10 +542,12 @@ put_data(all_vars_struct   *all_vars,
     }
     storage += out_data[OUT_SWE][0] + out_data[OUT_SNOW_CANOPY][0] +
                out_data[OUT_WDEW][0] + out_data[OUT_SURFSTOR][0];
-    out_data[OUT_WATER_ERROR][0] = calc_water_balance_error(inflow,
-                                                            outflow,
-                                                            storage,
-                                                            save_data->total_moist_storage);
+    out_data[OUT_WATER_ERROR][0] = calc_water_balance_error(
+        inflow,
+        outflow,
+        storage,
+        save_data->
+        total_moist_storage);
```

Comment:
this reformatting looks off to me

Reply:
Well, the original exceeded the 80-character limit, as far as I can tell, so it looks to me like the reformatting did the right thing in that respect...  How exactly should it look?

I could change it by hand but then uncrustify might attempt to change it again in future PRs...